### DISCUSSION TITLE
Lean vs agile vs design thinking for a design system in the Government of Canada?

### DISCUSSION THREAD

**Discussion by delisma** [alpha-canada-ca/gcds#9] [General]
Lean, agile, and design thinking are all methodologies that can be used to solve problems and improve processes. Lean is a methodology that is focused on maximizing value and minimizing waste. It is commonly used in manufacturing and production environments. Agile is a methodology that is focused on flexibility and collaboration. It is commonly used in software development. Design thinking is a methodology that is focused on user-centered design. It is used to solve complex problems and develop innovative solutions.

**Comment by delisma** [comment]
I think it really depends on the specific goals and constraints of this project. However, using a combination of lean, agile, and design thinking could be a good approach for building a design system if we understand the differences between all approached.

Lean methodology can be useful for identifying and eliminating waste in the design process, while agile methodology can help ensure that the design system is flexible and able to evolve over time. Design thinking can be used to understand the needs of the user and generate creative solutions to address those needs.

Ultimately, it may be necessary to tailor the methodology to the unique requirements of this project.

**Comment by chuqs** [comment]
Nice article and an eye opening one
